packages/approval-controller/src/ApprovalController.test.ts

```diff
@@ -109,6 +118,7 @@ describe('approval controller', () => {
           id: 'foo',
           origin: 'bar.baz',
           requestData: null,
+          requestState: null,
```

Comment:
Do we need to require `requestState` to always be defined? If we don't we can skip a breaking change

Reply:
Nevermind, I was reading this wrong.